### PR TITLE
Update and reword README.md

### PR DESCRIPTION
**Pull request by blag**
I noticed that the README hadn't been updated in awhile, so I tweaked some of the language, fixed a few grammatical issues, removed the broken Slack badge, and added a few more badges.

You can see the final rendered product by looking at the `update-readme` branch:
https://github.com/StackStorm/st2/tree/update-readme

TODO:

* [x] ~Tests~ (just kidding)
* [x] ~Update screenshot and linked YouTube video~
* [x] Update architecture diagram to remove Mistral

Edit: Updated TODO since I'm not going to have time to record a new YouTube video.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,14 @@
...
 
-[![Tests Build Status](https://travis-ci.org/StackStorm/st2.svg?branch=master)](https://travis-ci.org/StackStorm/st2) [![Packages Build Status](https://circleci.com/gh/StackStorm/st2/tree/master.svg?style=shield)](https://circleci.com/gh/StackStorm/st2) [![Codecov](https://codecov.io/github/StackStorm/st2/badge.svg?branch=master&service=github)](https://codecov.io/github/StackStorm/st2?branch=master) ![Python 2.7 | 3.6](https://img.shields.io/badge/python-2.7%20%7C%203.6-blue)  [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/1833/badge)](https://bestpractices.coreinfrastructure.org/projects/1833) [![Join our community Slack](https://stackstorm-community.herokuapp.com/badge.svg)](https://stackstorm.com/community-signup) [![Forum](https://img.shields.io/discourse/https/forum.stackstorm.com/posts.svg)](https://forum.stackstorm.com/)
+[![Apache Licensed](https://img.shields.io/github/license/StackStorm/st2)](LICENSE) 
+![Python 2.7 | 3.6](https://img.shields.io/badge/python-2.7%20%7C%203.6-blue) 
+[![Travis Unit Tests Status](https://travis-ci.org/StackStorm/st2.svg?branch=master)](https://travis-ci.org/StackStorm/st2)
```
Comment (armab) [inline README.md:7]: I think naturally all the CI build statuses come first as something that might change red/green.

Besides of that looks good.

Comment (blag) [reply README.md:5]: Good call.

**Comment by armab** [comment]
Following the https://github.com/StackStorm/st2/issues/4762#issuecomment-687476637, just changed the reference to the new diagram you shared.

@blag is there anything else we need to merge this?

**Review by armab — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -27,51 +34,51 @@ StackStorm is a platform for integration and automation across services and tool
...
 
-* **Rules** map triggers to actions (or to workflows), applying matching criterias and mapping trigger payload to action inputs.
+* **Rules** map triggers to actions (or to workflows), applying matching criterias and map trigger payload data to action inputs.
 
-* **Workflows** stitch actions together into “uber-actions”, defining the order, transition conditions, and passing the data. Most automations are more than one-step and thus need more than one action. Workflows, just like “atomic” actions, are available in action library, can be invoked manually or triggered by rules.
+* **Workflows** stitch actions together into "uber-actions", defining the order, transition conditions, and passing context data from one action to the next. Most automations are multi-step (eg: more than one action). Workflows, just like "atomic" actions, are available in the action library, and can be invoked manually or triggered by rules.
```
Comment (amanda11) [inline README.md:63]: Should that be eg. rather than eg:   ?

File: README.md [hunk 3]
```diff
@@ -27,51 +34,51 @@ StackStorm is a platform for integration and automation across services and tool
...
 
 ## Documentation
 
-Additional documentation describing installation proceduces, action/rule/workflow authoring, and how to setup and use triggers/sensors can be found at [StackStorm Docs](https://docs.stackstorm.com).
+Additional documentation, including installation proceduces, action/rule/workflow authoring, and how to setup and use triggers/sensors can be found at [https://docs.stackstorm.com](https://docs.stackstorm.com).
 
 ## Hacking / Contributing
 
-To set up dev environment and run StackStorm from sources, follow [these instructions](https://docs.stackstorm.com/development/sources.html).
+To set up development environment and run StackStorm from sources, follow [these instructions](https://docs.stackstorm.com/development/sources.html).
```
Comment (amanda11) [inline README.md:79]: Should it be "set up a development environment" - i.e. extra word a?

File: README.md [hunk 4]
```diff
@@ -85,7 +92,7 @@ in 48 hours or less. If further action is necessary, you may receive additional
 
 For more information, please refer to https://docs.stackstorm.com/latest/security.html
 
-## Copyright, License, and Contributors Agreement
+## Copyright, License, and Contributor Agreement
 
 Copyright 2020 The StackStorm Authors.
 Copyright 2014-2018 StackStorm, Inc.
```
Comment (amanda11) [inline README.md:98]: What was copyright for 2019? Its not in the list of years.

**Review by amanda11 — COMMENTED** [review]
LGTM - just had a few comments that you may or may not agree with.

File: README.md (hunk 2 above)
Comment (blag) [reply README.md:63]: Yep, TIL.

File: README.md (hunk 3 above)
Comment (blag) [reply README.md:79]: Yep, good catch!

File: README.md [hunk 5]
```diff
@@ -27,51 +34,51 @@ StackStorm is a platform for integration and automation across services and tool
...
 
-* **Rules** map triggers to actions (or to workflows), applying matching criterias and mapping trigger payload to action inputs.
+* **Rules** map triggers to actions (or to workflows), applying matching criterias and map trigger payload data to action inputs.
 
-* **Workflows** stitch actions together into “uber-actions”, defining the order, transition conditions, and passing the data. Most automations are more than one-step and thus need more than one action. Workflows, just like “atomic” actions, are available in action library, can be invoked manually or triggered by rules.
+* **Workflows** stitch actions together into "uber-actions", defining the order, transition conditions, and passing context data from one action to the next. Most automations are multi-step (eg: more than one action). Workflows, just like "atomic" actions, are available in the action library, and can be invoked manually or triggered by rules.
```
[the remark this answers is not in the document]
Comment (nzlosh) [reply README.md:63]: The correct form for the "For example" abbreviation is *e.g.*

**Comment by armab** [comment]
@blag There seems to be some very minor changes left to merge this doc update PR. It's part of the v3.3 Mistral Deprecation plan and I think would close the https://github.com/StackStorm/st2/issues/4762.

File: README.md (hunk 5 above)
Comment (nmaludy) [inline README.md:63]:
```suggestion
* **Workflows** stitch actions together into "uber-actions", defining the order, transition conditions, and passing context data from one action to the next. Most automations are multi-step (e.g. more than one action). Workflows, just like "atomic" actions, are available in the action library, and can be invoked manually or triggered by rules.
```

**Pull request merged**